src/Application.cc

```diff
@@ -524,6 +506,37 @@ std::string Application::DefaultConfigPath()
   return this->dataPtr->defaultConfigPath;
 }
 
+std::string Application::ResolveConfigFile(const std::string &_path)
```

Comment:
add separation line `/////////////////////////////////////////////////`

Reply:
Done in https://github.com/gazebosim/gz-gui/pull/631/commits/c2fa7925ee8c49e846957018dba9adc6b4a3202e